2022-02-08 17:17:03,120 INFO    MainThread:4798 [wandb_setup.py:_flush():71] setting env: {}
2022-02-08 17:17:03,121 INFO    MainThread:4798 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-08 17:17:03,121 INFO    MainThread:4798 [wandb_init.py:_log_setup():371] Logging user logs to /home/robotlabx/Desktop/DTAI/decnet2022/src/models/wandb/run-20220208_171703-2kl75qfm/logs/debug.log
2022-02-08 17:17:03,121 INFO    MainThread:4798 [wandb_init.py:_log_setup():372] Logging internal logs to /home/robotlabx/Desktop/DTAI/decnet2022/src/models/wandb/run-20220208_171703-2kl75qfm/logs/debug-internal.log
2022-02-08 17:17:03,121 INFO    MainThread:4798 [wandb_init.py:init():404] calling init triggers
2022-02-08 17:17:03,121 INFO    MainThread:4798 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-08 17:17:03,121 INFO    MainThread:4798 [wandb_init.py:init():460] starting backend
2022-02-08 17:17:03,121 INFO    MainThread:4798 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-08 17:17:03,169 INFO    MainThread:4798 [backend.py:ensure_launched():216] starting backend process...
2022-02-08 17:17:03,184 INFO    MainThread:4798 [backend.py:ensure_launched():222] started backend process with pid: 4853
2022-02-08 17:17:03,185 INFO    MainThread:4798 [wandb_init.py:init():469] backend started and connected
2022-02-08 17:17:03,188 INFO    MainThread:4798 [wandb_init.py:init():533] updated telemetry
2022-02-08 17:17:03,226 INFO    MainThread:4798 [wandb_init.py:init():563] communicating current version
2022-02-08 17:17:03,859 INFO    MainThread:4798 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-08 17:17:03,859 INFO    MainThread:4798 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-08 17:17:04,124 INFO    MainThread:4798 [wandb_init.py:init():606] starting run threads in backend
2022-02-08 17:17:05,598 INFO    MainThread:4798 [wandb_run.py:_console_start():1810] atexit reg
2022-02-08 17:17:05,598 INFO    MainThread:4798 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-02-08 17:17:05,598 INFO    MainThread:4798 [wandb_run.py:_redirect():1689] Redirecting console.
2022-02-08 17:17:05,599 INFO    MainThread:4798 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-08 17:17:05,599 INFO    MainThread:4798 [wandb_init.py:init():633] run started, returning control to user process
2022-02-08 17:17:05,913 INFO    MainThread:4798 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-08 17:17:05,914 INFO    MainThread:4798 [wandb_run.py:_restore():1752] restore
2022-02-08 17:17:08,113 INFO    MainThread:4798 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 790
}

2022-02-08 17:17:08,487 INFO    MainThread:4798 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 790
}

2022-02-08 17:17:09,128 INFO    MainThread:4798 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 790
}

2022-02-08 17:17:09,231 INFO    MainThread:4798 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 12133
}

2022-02-08 17:17:09,335 INFO    MainThread:4798 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 12133
}

2022-02-08 17:17:09,437 INFO    MainThread:4798 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 12133
}

2022-02-08 17:17:09,540 INFO    MainThread:4798 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12133
  total_bytes: 12133
}

2022-02-08 17:17:09,643 INFO    MainThread:4798 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12133
  total_bytes: 12133
}

2022-02-08 17:17:09,747 INFO    MainThread:4798 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12133
  total_bytes: 12133
}

2022-02-08 17:17:09,850 INFO    MainThread:4798 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12133
  total_bytes: 12133
}

2022-02-08 17:17:09,953 INFO    MainThread:4798 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12133
  total_bytes: 12133
}

2022-02-08 17:17:10,517 INFO    MainThread:4798 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12133
  total_bytes: 12133
}

2022-02-08 17:17:11,062 INFO    MainThread:4798 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12133
  total_bytes: 12133
}
local_info {
}

2022-02-08 17:17:12,139 INFO    MainThread:4798 [wandb_run.py:_append_files():2180] logging synced files
